meta-refkit/conf/bblayers.conf.sample

```diff
@@ -20,6 +20,7 @@ REFKIT_LAYERS = " \
   ##OEROOT##/../meta-security-isafw \
   ##OEROOT##/../meta-intel-realsense \
   ##OEROOT##/../meta-clang \
+  ##OEROOT##/../meta-flatpak \
```

Comment:
@mythi, @pohly So what LCONF_VERSION should be bumped to ?
5 or 8 ?
Or does this one go to 11 ?